src/app/components/content-finder-page/content-finder-page.component.spec.ts

```diff
@@ -42,61 +42,63 @@ describe('ContentFinderPageComponent', () => {
     component.selectedSubjects = ['Java', 'CSS'];
   });
 
-  /**
```

Comment:
Same issue, all comments have been removed. Some amount of comments/documentation is important.